cloudinit/config/cc_ssh.py

```diff
@@ -258,9 +260,19 @@ def handle(name: str, cfg: Config, cloud: Cloud, args: list) -> None:
         genkeys = util.get_cfg_option_list(
             cfg, "ssh_genkeytypes", GENERATE_KEY_NAMES
         )
+        # remove keys that are not supported in fips mode if its enabled
+        key_names = (
```

Comment:
It may be worth a LOG.debug mentioning the matched and skipped  FIPS_UNSUPPORTED_KEY_NAMES.

We could perform a `set(genkeys).difference(key_names)` to get that value for the log

Reply:
done.